roles/remediate/tasks/leapp_missing_pkg.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: leapp_missing_pkg | Install required leapp data package on public clouds
+  vars:
+    entry_title_pattern: Package "[-\w]+" is missing
+    leapp_report_location: /var/log/leapp/leapp-report.json
+  block:
+    - name: leapp_missing_pkg | Check that the leapp-report.json exists
+      ansible.builtin.stat:
+        path: "{{ leapp_report_location }}"
+      register: leapp_report_stat
+
+    - name: leapp_missing_pkg | End play if no leapp report exists
+      ansible.builtin.meta: end_play
+      when: leapp_report_stat.stat.exists is false
+
+    - name: leapp_missing_pkg | Read leapp report
+      ansible.builtin.slurp:
+        src: "{{ leapp_report_location }}"
+      register: leappreport
+
+    - name: leapp_missing_pkg | Parse leapp report to json
+      ansible.builtin.set_fact:
+        leappreportdata: "{{ leappreport.content | b64decode | from_json }}"
+
+    - name: leapp_missing_pkg | Find matching entries
+      ansible.builtin.set_fact:
+        remediation: "{{ item.detail.remediations | selectattr('type', 'eq', 'command') | first }}"
+      loop: "{{ leappreportdata.entries }}"
+      when: item.title is match(entry_title_pattern) and (item.detail.remediations | selectattr('type', 'eq', 'command') | list | length > 0)
+
+    - name: leapp_missing_pkg | End execution of playbook if no entry found in leapp report
+      ansible.builtin.meta: end_play
```

Comment:
I noticed some places we are doing `end_play` and others we are using `end_host`. Shouldn't we use `end_host` everywhere?